src/OrchardCore.Modules/OrchardCore.Queries/Views/Admin/Query.cshtml

```diff
@@ -3,7 +3,7 @@
 @inject IContentManager ContentManager
 
 @{
-    var matchAllQuery = Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(@"SELECT * FROM ContentItemIndex"));
+  var matchAllQuery = OrchardCore.Queries.Sql.ViewModels.AdminQueryViewModel.MatchAllQueryBase64;
```

Comment:
```suggestion
  var matchAllQuery = AdminQueryViewModel.MatchAllQueryBase64;
```

Reply:
It's not part of the usings

Reply:
Sorry, I mean to use the model, I updated the above suggestion

Reply:
Static properties can't be accessed from instance references.